src/components/HelloWorld.vue

```diff
@@ -11,7 +11,9 @@ export default {
   name: "HelloWorld",
   methods: {
     logOut() {
-      this.$store.dispatch(AUTH_ACTIONS.LOG_OUT, this.$router);
+      this.$store
+        .dispatch(AUTH_ACTIONS.LOG_OUT, this.$router)
+        .then(this.$router.push("/login"));
```

Comment:
```suggestion
        .then(() => this.$router.push("/login"));
```

Reply:
I nemoj slati router u akcije, nije bas dobra praksa da to uradimo iz komponente :D